torchaudio/functional.py

```diff
@@ -801,3 +801,21 @@ def compute_deltas(specgram, win_length=5, mode="replicate"):
     return torch.nn.functional.conv1d(
         specgram, kernel, groups=specgram.shape[1] // specgram.shape[0]
     ) / denom
+
```

Comment:
Have you checked for `flake8` on the PR? It's a good way to see if there's lint to correct.